Week 1/Weekly Practice Problems (Week 1)/2413: Smallest Even Multiple/dominic.cpp

```diff
@@ -0,0 +1,6 @@
+class Solution {
+public:
+    int smallestEvenMultiple(int n) {
+        return n%2 ? n*2 : n;
```

Comment:
```suggestion
        return n << (n&1);
```
to make it even cleaner :)

Reply:
Ah, bit shifting...fair enough :P 
I had to think about how this works, but wow. It shifts 1 if n ends with 1, and shifts 0 otherwise. Clever :o